aws-cloud-integrations.md

```diff
@@ -504,16 +504,15 @@ It may also be required to remove and reinstall Kubecost. If doing this please r
 
 ## Spot Data feed integration
 
-Kubecost will reconcile your spot prices with CUR billing reports as they become available (usually 1-2 days), but pricing data can be pulled hourly by integrating directly with the AWS spot feed. To enable, follow these steps:
+The Spot Data feed integration provides the same functionality as the AWS Cloud Integration, or CUR. The only difference is that the spot prices are updated hourly with the Spot Feed Integration, while the CUR can be delayed up to 48 hours. Therefore, if comprehensive prices are required and hourly spot prices are not, only the CUR integration is necessary.
```

Comment:
Even though this PR is just moving an existing paragraph to a new location, IMO it would be a good time to reword these statements to make them more clear. It's a bit confusing to say the Spot data feed provides the same functionality as the CUR as that's not entirely true.

Example:
```suggestion
While AWS updates its CUR multiple times per day, it can be delayed up to 48 hours. Kubecost's Spot instance data feed integration will enable you to receive accurate cost estimates for your Spot nodes without the delay of waiting for the cost data to appear in the CUR.

> **Note**:  This integration is optional. For the best experience, enable both the CUR and Spot instance data feed integrations.
```